### ISSUE TITLE
nightly: SIGSEGV on `api::get_runtime_file` only somewhat reproducible with test

### ISSUE THREAD

**Issue by ModProg** [noib3/nvim-oxi#45]
When calling `api::get_runtime_file` I get a SIGSEV from neovim. (though calling it from lua is fine).

This is only somewhat reproducible, I added a test like this: 
```rs
#[oxi::test]
fn get_runtime_file() {
    api::get_runtime_file("test", true).unwrap();
    panic!();
}
```

And it did not reach the panic, showing as "successful".

On stable neovim it worked fine.

The underlying issue might be on the neovim side and outside this repo, but the test should fail IMO.

**Comment by noib3** [comment]
Yes tests on nightly are kinda flaky. I've experienced the same issue before of tests containing `panic!`s passing as successfull. Also the nightly CI is failing bc of a few tests which fail with unexpected error messages which I haven't been able to debug yet. Not really sure what's causing all of this tbh. Further investigation is definitely required.

**Comment by noib3** [comment]
fixed in 07b03d3fb6d46862111c865779d053bad5bce0d0
